Solution Week-1/J1P2.py

```diff
@@ -1,8 +1,10 @@
 #2.Write a python script to enter any number and print the sum of its digit.
-def sum():

```

Comment:
what is use of def?

Reply:
'def' is a keyword that is used to create a user defined function(UDF)...
Thank you..